articles/architecture-scenarios/implementations/mobile-api/api-implementation-nodejs.md

```diff
@@ -0,0 +1,7 @@
+---
+description: The Node.js implementation of the API for the Mobile + API architecture scenario
+url: /architecture-scenarios/application/mobile-api/api-implementation-nodejs
+toc: true
+---
+
```

Comment:
There is something wrong with the metadata format, it's displayed like that:

![image](https://user-images.githubusercontent.com/11715799/29662238-3febb646-88cf-11e7-8d14-f7cd327f2f2c.png)